docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,30 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a zone is mechanically cooled or not.  DOES NOT CHECK FOR TRANSFER AIR
+
+**Inputs:**
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMD,zone)`
+	- loop through each heating system: `for system in list_hvac_systems:`
+		- look for a cooling system: `if system_p.cooling_system != "Null":`
+			- check to see if the cooling_system_options equals "None": `if system_p.cooling_system.cooling_system_options != "None":`
+				- there is a cooling system in the zone: `result = TRUE`
```

Comment:
I am not clear on the definition of mechanically cooled in this clause.
When defining whether a zone is conditioned or not, one threshold is cooling capacity shall be greater than 3.4 Btu/(hr * ft2). Does the threshold applies here too?

If so, it will resolve the question in the Note.

Reply:
that's a good question.  G3.1.1.e also mentions zones that receive transfer air from mechanically cooled zones, so I am led to believe that it's a zone with any mechanical cooling capability.  Probably a question for @mkarpman

Reply:
The distinctions that you discuss above all apply to definition of conditioned/indirectly conditionsed/semiheated spaces so don't think they apply here. I would take the language of this exception literaly - a zone should have a heating system but no cooling system in P-RMD in order to meet this exception.

Reply:
I updated the function and renamed it for clarity to `is_zone_mechanically_heated_and_not_cooled`